source/includes/_account.md

```diff
@@ -2169,7 +2166,7 @@ List<String> NULL_PLUGIN_NAMES = null;
 Map<String, String> NULL_PLUGIN_PROPERTIES = null;
 
 Payment payment = accountApi.processPaymentByExternalKey(authTransaction, 
-													              externalKey, 
+													                            externalKey, 
 		                                                  paymentMethodId, 
```

Comment:
The `externalKey` parameter is already specified ( see https://killbill.github.io/slate/?java#account-trigger-a-payment-authorization-purchase-or-credit-using-the-account-external-key), any particular reason for adding it again? Also, the indentation is a bit off:

<img width="376" alt="image" src="https://github.com/killbill/slate/assets/85998496/52019808-51db-46cf-a0e8-949bc4ee1e9a">

Reply:
![image](https://github.com/killbill/slate/assets/75751774/8788e277-baa6-4c33-825c-0e8e73566c62)

![image](https://github.com/killbill/slate/assets/75751774/903b99d7-be3a-41b0-9d02-d6cd1a28bff7)

Hi Reshma, not sure why it is showing externalKey changed, but I have it once in the request. Tried to fix the indentation also.